plugins/arDominionB5Plugin/modules/admin/templates/error404Success.php

```diff
@@ -0,0 +1,20 @@
+<section class="w-50 mx-auto rounded border border-dark mt-3 border-1 bg-white">
```

Comment:
I'm wondering if [`role="alert"`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_alert_role) would be a good fit in these error pages.